File: converted/src/zombie/core/znet/IServerBrowserCallback.cpp
#include "zombie/core/znet/IServerBrowserCallback.h"

namespace zombie {
namespace core {
namespace znet {

void IServerBrowserCallback::OnServerResponded(int var1) {
  // TODO: Implement OnServerResponded
}

void IServerBrowserCallback::OnServerFailedToRespond(int var1) {
  // TODO: Implement OnServerFailedToRespond
}

void IServerBrowserCallback::OnRefreshComplete() {
  // TODO: Implement OnRefreshComplete
}

void IServerBrowserCallback::OnServerResponded(const std::string &var1,
                                               int var2) {
  // TODO: Implement OnServerResponded
}

void IServerBrowserCallback::OnServerFailedToRespond(const std::string &var1,
                                                     int var2) {
  // TODO: Implement OnServerFailedToRespond
}

void IServerBrowserCallback::OnSteamRulesRefreshComplete(
    const std::string &var1, int var2) {
  // TODO: Implement OnSteamRulesRefreshComplete
}

} // namespace znet
} // namespace core
} // namespace zombie

File: source/zombie/core/znet/IServerBrowserCallback.java
// Decompiled on Sat Jan 17 08:24:00 EST 2026 with Zomboid Decompiler v0.2.3 using Vineflower.
package zombie.core.znet;

public interface IServerBrowserCallback {
    void OnServerResponded(int var1);

    void OnServerFailedToRespond(int var1);

    void OnRefreshComplete();

    void OnServerResponded(String var1, int var2);

    void OnServerFailedToRespond(String var1, int var2);

    void OnSteamRulesRefreshComplete(String var1, int var2);
}
